docs/guides/removing-players.md

```diff
@@ -0,0 +1,38 @@
+RemovingPlayers
+===============
+
+Sometimes, you want to remove players after page load (in single page apps, for instance). It's easy to manage, but there are some simple rules you need to follow.
+
+Call `.dispose()`
+-----------------
+
+Before removing the html associated with your videojs player from the page always call the player's `dispose()` method
+
+```javascript```
+var oldPlayer = document.getElementById('my-player');
+videojs(oldPlayer).dispose(); // Don't skip this step!
+oldPlayer.parentNode.removeChild(oldPlayer);
+```
+
+Or...Use Unique Ids
+-------------------
+
+If you prefer not to call `dispose()` on a player, you can always create new players with unique ids for that page load.
```

Comment:
I'd probably suggest just removing this bit altogether. Any time you remove a player from the DOM (or hide it, or just don't need it anymore), it's a good idea to remove it, even if just from a resource management perspective.